testsuite/features/step_definitions/common_steps.rb

```diff
@@ -1413,10 +1413,69 @@ def server_secret
 And(/^I check the Cobbler parameter "([^"]*)" with value "([^"]*)" in the isolinux.cfg$/) do |param, value|
   tmp_dir = "/var/cache/cobbler/buildiso"
   result, code = $server.run("cat #{tmp_dir}/isolinux/isolinux.cfg | grep -o #{param}=#{value}")
-  raise "error during veryfying isolinux.cfg parameter for Cobbler buildiso.\nLogs:\n#{result}" if code.nonzero?
+  raise "error during verifying isolinux.cfg parameter for Cobbler buildiso.\nLogs:\n#{result}" if code.nonzero?
 end
 
 When(/^I cleanup after Cobbler buildiso$/) do
   result, code = $server.run("rm -Rf /var/cache/cobbler")
   raise "error during Cobbler buildiso cleanup.\nLogs:\n#{result}" if code.nonzero?
 end
+
+When(/^I reboot server through SSH$/) do
+  node = get_target('server')
+  temp_server = twopence_init("ssh:#{$server.public_ip}")
+  temp_server.extend(LavandaBasic)
+  temp_server.run("reboot > /dev/null 2> /dev/null &")
```

Comment:
use single quotes

why not simply : `$server.run('reboot > /dev/null 2> /dev/null &')` ? (I'm honestly asking, maybe there's a very good reason)

Reply:
This step is also used when rebooting while the server has a different hostname which is unknown by the controller, and if I am not mistaken `$server` uses the hostname, that is why I created a temporary node with SSH using the public IP